### PR TITLE
update instructions

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1193

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/commerce/manage-saas-apps.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ ms.date: 05/31/2019
 
 # Manage third-party app subscriptions for your organization
 
-You can manage licenses and billing for third-party apps in Microsoft 365 admin center with preview mode turned on. Updated features include enhanced subscription management, improved access to billing information, and improved flexibility for managing bills. Subscription management is based on Microsoft’s updated commerce platform. This applies to software-as-a-service apps that customers purchase directly, or from third-party provider.
+You can manage licenses and billing for third-party apps in new Microsoft 365 admin center. Updated features include enhanced subscription management, improved access to billing information, and improved flexibility for managing bills. Subscription management is based on Microsoft’s updated commerce platform. This applies to software-as-a-service apps that customers purchase directly, or from third-party provider.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
You can manage licenses and billing for third-party apps in the new Microsoft 365 admin center. Updated features include enhanced subscription management, improved access to billing information, and improved flexibility for managing bills. Subscription management is based on Microsoft’s updated commerce platform. This applies to software-as-a-service apps that customers purchase directly, or from a third-party provider.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ Support for third-party apps is available in these regions:
 Admins need to activate third-party apps before assigning them to users. These apps are activated in the third-party publisher’s portal. 
 
 **To activate third-party apps**
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that the **Try the new admin center** toggle is turned on.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 3]
```diff
@@ -50,7 +50,8 @@ Support for third-party apps is available in these regions:
 Admins need to activate third-party apps before assigning them to users. These apps are activated in the third-party publisher’s portal. 
 
 **To activate third-party apps**
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center 
+ ** toggle is turned on.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 4]
```diff
@@ -73,7 +74,8 @@ Admins need to activate third-party apps before assigning them to users. They're
 
 **To assign third-party apps**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that the **Try the new admin center** toggle is turned on.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 5]
```diff
@@ -73,7 +74,8 @@ Admins need to activate third-party apps before assigning them to users. They're
 
 **To assign third-party apps**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center 
+ ** toggle is turned on.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 6]
```diff
@@ -83,7 +85,8 @@ Admins can change the number of licenses owned by their organization. This only
 
 **To change license quantity**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that the **Try the new admin center** toggle is turned on.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 7]
```diff
@@ -83,7 +85,8 @@ Admins can change the number of licenses owned by their organization. This only
 
 **To change license quantity**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center 
+ ** toggle is turned on.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 8]
```diff
@@ -99,7 +102,8 @@ For more information on billing profiles, see [Understand billing profiles](http
 
 **To change the billing profile on a software-as-a-service app subscription**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that the **Try the new admin center** toggle is turned on.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 9]
```diff
@@ -99,7 +102,8 @@ For more information on billing profiles, see [Understand billing profiles](http
 
 **To change the billing profile on a software-as-a-service app subscription**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center 
+ ** toggle is turned on.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 10]
```diff
@@ -111,7 +115,8 @@ Customers can cancel software-as-a-service app from the app page.
 
 **To cancel software-as-a-service app subscription**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that the **Try the new admin center** toggle is turned on.
```

File: microsoft-365/commerce/manage-saas-apps.md [hunk 11]
```diff
@@ -111,7 +115,8 @@ Customers can cancel software-as-a-service app from the app page.
 
 **To cancel software-as-a-service app subscription**
 
-1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and turn on **Try the preview**.
+1. Go to [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890), and make sure that **Try the new admin center 
+ ** toggle is turned on.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-saas-apps.md]:
```suggestion
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by mypil** [comment]
@TrudyHa - PR has been copy edited and is ready for final review. This is based on issue #1193.

Thank you.

cc: @kenwith

**Pull request merged**